content/participate/review-changes.adoc

```diff
@@ -8,21 +8,30 @@ tags:
   - review
 ---
 
-NOTE: This page is under development, there will be more content added soon.
-See the jira:WEBSITE-662[] EPIC for tasks related to this page, contributions are welcome!
-
 Help us to review changes to code or documentation!
+All pull requests within Jenkins GitHub organizations are publicly open for reviews, and any reviews will be much appreciated.
 
 == Code Reviews and Copy Editing
 
-You can help review changes to the code and the documentation through following lists:
+Contributors and component maintainers may explicitly request request reviews from other community members.
+Below there are some queries which can help to find such open pull requests where help is needed:
 
-- link:https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+team%3Ajenkinsci%2Fcode-reviewers[Open pull requests for code reviews]
-- link:https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+team%3Ajenkins-infra%2Fcopy-editors[Open pull requests for copy editing]
+- link:https://github.com/search?q=is%3Aopen+is%3Apr+team-review-requested%3Ajenkins-infra%2Fcopy-editors[Open pull requests for copy editing], mostly for Jenkins website
+- link:https://github.com/search?q=is%3Aopen+is%3Apr+%22jenkinsci%2Fcode-reviewers%22&type=Issues[Open pull requests for code reviews] (https://github.com/orgs/jenkinsci/teams/code-reviewers[@jenkinsci/code-reviewers] team is mentioned)
+- link:https://github.com/search?q=is%3Aopen+is%3Apr+%22jenkinsci%core-pr-reviewers%22&type=Issues[Open pull requests for code reviews in the Jenkins Core] (https://github.com/orgs/jenkinsci/teams/core-pr-reviewers[@jenkinsci/core-pr-reviewers] team is mentioned)
```

Comment:
This query URL is not working for me.  I'm not sure how to express it myself.  Could you double check that it works for you?